crates/sui-node/src/admin.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
This is kind of ad-hoc, so I'd add a line or two for what we want this to accomplish.